ush/config_defaults.yaml

```diff
@@ -1066,11 +1091,18 @@ rrfs:
   #
   #-----------------------------------------------------------------------
   #
+  # Set parameters associated with running retrospective experiments.  Definitions:
+  #
+  # DO_RRFS_DEV:
+  # Flag to turn on RRFS dev workflow. Should be removed
+  # in the future once SRW and RRFS dev workflows are completely integrated
+  #
```

Comment:
I don't think this should be included at this time, especially with the caveat that it should be removed. Why not just set `NEED_ALL_LBCS` to what it should be?

Reply:
The 0th boundary condition is set only for RRFS not for SRW or AQM, so we need to tell it when it is RRFS that is running. The `DO_RRFS_DEV` is also used in other places where things are differently for RRFS so I think it is good to have it. Otherwise I will have to define NEED_ALL_LBCS and other similar flags in `config_defaults.yaml` when one DO_RRFS_DEV will suffice for all things that are done differently in RRFS.